new-pipeline/dist.html

```diff
@@ -55,7 +55,10 @@ <h2 style="font-size: 18px; margin-bottom: 0">
                 </h2>
                 <hr></hr>
                 <figure class="col-md-12">
-                  <figcaption id="dist-caption"></figcaption>
+                <div class="caption-div">
```

Comment:
This `div` is missing one indentation level. Then the `figcaption` need to move along accordingly.